src/ape/api/accounts.py

```diff
@@ -19,10 +19,19 @@
 
 # NOTE: AddressAPI is a dataclass already
 class AccountAPI(AddressAPI):
+    """
+    A base account API with inherited address traits.
```

Comment:
They will see the inheritance in the docs already

```suggestion
    An API class representing an account.
```